modular_citadel/code/modules/client/loadout/__donator.dm

```diff
@@ -367,4 +367,18 @@ datum/gear/darksabresheath
 	name = "Soul Necklace"
 	category = SLOT_NECK
 	path = /obj/item/clothing/neck/undertale
-	ckeywhitelist = list("twilightic")
\ No newline at end of file
+	ckeywhitelist = list("twilightic")
+
+/datum/gear/telegramsuit
+	name = "Singing Telegram Suit"
+	category = SLOT_IN_BACKPACK
+	cost = 0
+	path = /obj/item/clothing/head/telegramhat
+	ckeywhitelist = list("TrilbySpaseClone")
```

Comment:
wasn't it `TrilbySpaceClone` ? Make sure you wrote down your own ckey correctly.

Reply:
That is my ckey